dist/index.js

```diff
@@ -0,0 +1,2575 @@
+/******/ (() => { // webpackBootstrap
```

Comment:
Is this code file generated? If so, can you add some documentation as to how it can be re-generated? My assumption is that it is based on `@vercel/ncc`.

Reply:
> I need to do some research around `@vercel/ncc`, and how it works. Overall this looks awesome. If you have any quick links about the tech handy, then I'd be happy to look them over.

Nothing outside the documentation. As mentioned before, I came across it after studying some of GitHub's own actions. Many (if not all) seem to make use of it.

> Is this code file generated? If so, can you add some documentation as to how it can be re-generated?

I'm actually wondering if one should go as far and re-generate the dist folder automatically. It's easy with packages published on npm, I'm not sure which is the best method in this case. Using [`husky`](https://github.com/typicode/husky) comes to mind first.

Reply:
If you want to toss in the command in `README.md` documentation, or hook up husky I could support either one.

Reply:
I went with a documentation for now, since I wasn't fully satisfied with the husky workflow.

Reply:
Awesome work - Thanks for all of your help.